packages/documentation/src/components/dashboard/DashboardDataFetch.jsx

```diff
@@ -1,3 +1,34 @@
+import { Octokit } from '@octokit/rest';
+
+function getWorkflowRunsAsCommitStatusObject(repo) {
+  const octokit = new Octokit();
+
+  const params = {
+    owner: 'department-of-veterans-affairs',
+    repo,
+    workflow_id: 'continuous-integration.yml',
+    branch: 'main',
+    per_page: 30,
```

Comment:
One thing I'm not 100% sure of: if a workflow is re-run for a given commit, will that count as multiple runs here? Or does it only return the final run per commit (30 commits is 30 API results)?

Reply:
I want to say it's the final run because I think it keeps the same `id` even when it's re-run, but I am not 100% sure on that. I had to deal with that situation back when I worked on the CI GHA.

Reply:
Yeah I just manually re-ran my CI workflow for this PR and it kept its `id` and stayed in the same position in the response: https://api.github.com/repos/department-of-veterans-affairs/veteran-facing-services-tools/actions/workflows/continuous-integration.yml/runs

Reply:
Okay great, we should be good with 30 per page then. Thanks for checking!